.circleci/config.yml

```diff
@@ -515,7 +515,7 @@ jobs:
             gsutil cp gs://all-of-us-workbench-test-credentials/.npmrc .
             yarn install
             yarn codegen
-            REACT_APP_ENVIRONMENT=pr-site-$PR_SITE_NUM yarn run build --aot --no-watch --no-progress
+            REACT_APP_ENVIRONMENT=test yarn run build --aot --no-watch --no-progress
```

Comment:
If we go with this approach, I will delete the unused PR site configuration files.